src/components/steps/WelcomeForm.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div data-test="welcome-back" class="welcome-back">
     <div class="bar">
-      <CustomIcon class-name="row sheikah-icon" name="SheikahIcon" />
+      <CustomIcon class-name="row sheikah-icon" name="sheikah-icon" />
```

Comment:
In the FTU view, the buttons are not aligned

![image](https://github.com/witnet/sheikah/assets/24905681/3c82816d-49eb-402a-aff7-e0c50785692c)